src/main/java/nl/cherement/jak/service/BoardService.java

```diff
@@ -12,14 +14,28 @@
 public class BoardService extends AbstractService<BoardEntity> {
 
     @Autowired
-    BoardRepository repository;
+    BoardRepository boardRepository;
+
+    @Autowired
+    UserRespository userRespository;
 
     public BoardService(BoardRepository repository) {
         super(repository);
     }
 
     public List<BoardEntity> findByUserName(String username) {
-        return repository.findByUsers_Username(username);
+        return boardRepository.findByUsers_Username(username);
+    }
+
+    public void addUser(Long boardId, Long userId) {
```

Comment:
Wordt deze methode nooit gebruikt?

Reply:
Wacht wtf de methode waar deze werd gebruikt heeft @schaapkabap net weggehaald. wtf is going on